### ISSUE TITLE
"Warning: Invalid ARIA attribute `ariaHidden`. Did you mean `aria-hidden`?" when using HiSpeakerWave

### ISSUE THREAD

**Issue by ethandaniel** [react-icons/react-icons#642]
**Describe the bug**
when using HiSpeakerWave the warning in the title appears.

**Expected behavior**
In the svg of HiSpeaker wave, the tag 'ariaHidden="true"' should be 'aria-hidden="true"'

aria is an exception to the camelCase rule. Look at the documentation here -- [Official React Docs](https://reactjs.org/docs/accessibility.html#wai-aria)

there are probably more icons with this error, but I'm using the HiSpeaker one, which is how I found the error

**Comment by VitalyKrenel** [comment]
Confirmed that there are at least one more component with this error: getting same error on `HiOutlineDocumentArrowUp`
<img width="505" alt="image" src="https://user-images.githubusercontent.com/17354709/201393926-ebabff70-be45-4d3a-a964-0f2708d9c079.png">

**Comment by gustavodamazio** [comment]
This bug also occurs to me.
```
Warning: Invalid ARIA attribute `ariaHidden`. Did you mean `aria-hidden`?
```
![image](https://user-images.githubusercontent.com/52186091/201445622-1b10425a-fbe8-46ea-a4b7-f1c812ef4dba.png)

**Comment by gustavodamazio** [comment]
I created a pull request to solve the problem, vote on the [fix(logic-convertIconData): ignore camelCase in aria-* attributes](https://github.com/react-icons/react-icons/pull/643) for the maintainers to review.

**Comment by skydive-missile** [comment]
Hope the CiIcon and Hi2Icon fix soon!

**Comment by kamijin-fanta** [comment]
fixed in https://github.com/react-icons/react-icons/pull/630
